core/src/interface/ICoreContract.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
+
+import {IExtensionTypes} from "./IExtensionTypes.sol";
+
+interface ICoreContract is IExtensionTypes {
+    /*//////////////////////////////////////////////////////////////
+                                STRUCTS
+    //////////////////////////////////////////////////////////////*/
+
+    struct CallbackFunction {
+        uint256 callbackFunctionBitflag;
```

Comment:
`callback` is redundant, we're already defining this in the `CallbackFunction` struct.

nit: Bitflags

Reply:
> nit: Bitflags

This is meant to be singular bit _flag_, since the value expected here is 2 ** n for n >= 1.